core/unit_test/TestAtomicOperations.hpp

```diff
@@ -536,6 +536,8 @@ bool DivAtomicTest( T i0, T i1 )
 
   bool passed = true;
 
+  using std::abs;
+  using Kokkos::abs;
```

Comment:
ADL should suffice for picking up the right `abs` function, no?  The following builds just fine:
```c++
#include <complex>

namespace Foo {
    template<class T>
    struct complex {
        T re, im;
    };

    template<class T>
    T abs (const complex<T>& x) {
        return std::sqrt (x.re * x.re + x.im * x.im);
    }
}


int main() {
    std::complex<double> x (2.0, 3.0);
    const double x_abs = abs (x);
    Foo::complex<double> y {4.0, 5.0};
    const double y_abs = abs (y);
    return 0;
}
```

Reply:
Yes, due to ADL it isn't neccessary to import `Kokkos::abs`. I just thought this documents better that we expect either `std::abs` or `Kokkos::abs` here.

Reply:
If someone were to overload `Kokkos::abs` to take `std::complex<T>`, then the `using` statements would result in a build error:
```c++
#include <complex>

namespace Foo {
    template<class T>
    struct complex {
        T re, im;
    };

    template<class T>
    T abs (const complex<T>& x) {
        return std::sqrt (x.re * x.re + x.im * x.im);
    }

    template<class T>
    T abs (const std::complex<T>& x) {
        return std::sqrt (std::real(x) * std::real(x) + std::imag(x) + std::imag(x));
    }
}

int main() {
    // Uncommenting the following line results in a build error.
    // using Foo::abs;
    
    std::complex<double> x (2.0, 3.0);
    const double x_abs = abs (x);
    Foo::complex<double> y {4.0, 5.0};
    const double y_abs = abs (y);
    return 0;
}
```

Reply:
Anyway no biggie :)